exercises/diamond/example.rs

```diff
@@ -0,0 +1,63 @@
+static ABC: &'static str = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
+pub fn get_diamond(c: char) -> Vec<String> {
+    if c == 'A' {
+        return vec![String::from("A")];
+    }
+    let mut result: Vec<String> = Vec::new();
+
+    //build first half
+    for e in ABC.chars() {
+        result.push(get_line(e, c).clone());
+        if e == c {
+            break;
+        }
+    }
+
+    //build second half
+    let mut rev = result.clone();
+    rev.pop(); //remove middle pice to avoid duplicates
+    for e in rev.iter().rev().into_iter() {
+        result.push((*e).clone());
```

Comment:
You could probably avoid this second clone by using [`drain`](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.drain) on `rev`.